scripts/visum.sh

```diff
@@ -1,14 +1,43 @@
-if [ "$1" == "" ]; then
-  echo "No file path provided."
+print_error() {
+  >&2 echo $@
+}
+
+if [[ "$1" == ""  ]]; then
+  print_error "No file path provided."
+  exit 1
+fi
+if [ ! -f "$1" ]; then
+  print_error "$1: No such file or directory"
+  exit 1
+fi
+if [ ! -r "$1" ]; then
+  print_error "$1: Permission denied"
   exit 1
 fi
 
+is_office_file() {
+	file_format=`file $1 | rev | cut -d':' -f1 | rev`
+	supported_formats=( word excel composite microsoft powerpoint )
+
+	# verify file format is one of supported_formats
+	echo "${file_format,,}" | grep -qE "\b(`local IFS="|"; echo "${supported_formats[*]}"`)\b"
+	echo $?
+}
+
+FILEPATH=`realpath $1`
+if [ "$(is_office_file $FILEPATH)" == 1 ]; then
+	print_error "$1: Invalid file format"
+	exit 1
+fi
+
+# hide further errors
+exec 2>/dev/null
```

Comment:
Could you also explain this line?

Reply:
I updated the README.md to go with this change. For CLI usage, you used to recommend redirecting stdout and stderr to /dev/null. However, doing so would prevent error messages from showing up.
Therefore, I would recommend not to redirect stderr to /dev/null in the visum for the whole visum script (via provided bash function), but only redirect errors eventually coming after error handling. This would let error handling messages show up.

Of course, if you deem it unrelevant, I can remove it!